lib/CLI/Types.hs

```diff
@@ -73,7 +73,7 @@ data CopyResult
   = CPRSuccess [(QualifiedPath EntryPath, QualifiedPath EntryPath)]
   | CPRPathNotFound (QualifiedPath Path)
   | CPRMissingEntryName
-  | CPRSamePath Path Path
+  | CPRSamePath (QualifiedPath Path) Path
```

Comment:
Why not `CPRSamePath (QualifiedPath Path) (QualifiedPath Path)`?

Reply:
Removed second argument